client/my-sites/plans-v2/utils.ts

```diff
@@ -416,3 +418,43 @@ export function isUpgradeable( slug: string ): boolean {
 export function getProductUpsell( slug: string ): string | null {
 	return UPSELL_PRODUCT_MATRIX[ slug ];
 }
+
+/**
+ * Append "Available Options: Real-time and Daily" to the product description,
+ * if product/plan offers 'realtime' and 'daily' options, AND product/plan is not already owned,
+ *
+ * @param product SelectorProduct
+ * @param ownedProducts array | string
+ *
+ * @returns ReactNode | TranslateResult
+ */
+export const getJetpackDescriptionWithOptions = (
+	product: SelectorProduct,
+	ownedProducts: string[] | string | null
+): React.ReactNode | TranslateResult => {
+	if ( ! ownedProducts ) {
+		return product.description;
+	}
+
+	const productsOwned =
+		'item-type-plan' === product.type && 'string' === typeof ownedProducts
```

Comment:
Instead of using the string directly we can use the constant `ITEM_TYPE_PLAN` that is in `client/my-sites/plans-v2/constants.ts`

Reply:
And since we don't need to check if the product is "owned" (discussed [above](https://github.com/Automattic/wp-calypso/pull/45197#discussion_r477318896)) , then I can remove this `productsOwned` alltogether.